bundle/manifests/grafana-operator.clusterserviceversion.yaml

```diff
@@ -94,331 +94,323 @@ metadata:
+                    memory: 550Mi
+                  requests:
+                    cpu: 100m
+                    memory: 20Mi
                 securityContext:
-                  runAsNonRoot: true
-                serviceAccountName: grafana-operator-controller-manager
-                terminationGracePeriodSeconds: 10
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: grafana-operator-controller-manager
+              terminationGracePeriodSeconds: 10
       permissions:
-        - rules:
-            - apiGroups:
-                - ""
-              resources:
-                - configmaps
-              verbs:
-                - get
-                - list
-                - watch
-                - create
-                - update
-                - patch
-                - delete
-            - apiGroups:
-                - coordination.k8s.io
-              resources:
-                - leases
-              verbs:
-                - get
-                - list
-                - watch
-                - create
-                - update
-                - patch
-                - delete
-            - apiGroups:
-                - ""
-              resources:
-                - events
-              verbs:
-                - create
-                - patch
-          serviceAccountName: grafana-operator-controller-manager
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: grafana-operator-controller-manager
     strategy: deployment
   installModes:
-    - supported: true
-      type: OwnNamespace
-    - supported: true
-      type: SingleNamespace
-    - supported: true
-      type: MultiNamespace
-    - supported: true
-      type: AllNamespaces
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: true
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
   keywords:
-    - Grafana
-    - Metrics
-    - Observability
+  - Grafana
+  - Metrics
+  - Observability
   links:
-    - name: Grafana Operator
-      url: https://grafana-operator.github.io/grafana-operator
-  maturity: alpha
+  - name: Grafana Operator
+    url: https://grafana.github.io/grafana-operator
+  maturity: stable
   minKubeVersion: 1.23.0
   provider:
     name: Community
-  relatedImages:
```

Comment:
So we don't use the bundle folder any more in this repo after https://github.com/grafana/grafana-operator/pull/1234 but  it's nice to generate it one last time to make it easier when we do releases to the other repo. 
But you need to run `make bundle/redhat` to get the image SHAs so OLM works in disconnected mode.

Reply:
Having the generated bundle in this repo made it easier for me to verify my changes without having to copy the bundle across locations every time. I think it could make sense to at least track minor updates in this repo as a sanity check for future releases.

What do you think?

Reply:
Sorry I thought you had deleted `relatedImages` all together. I think we are thinking the same way.